### ISSUE TITLE
Absolute paths in meta files

### ISSUE THREAD

**Issue by mihai8804858** [spotify/XCRemoteCache#36]
Hi,

I am seeing some absolute paths in the uploaded meta files. The paths are part of the `.../Build/Intermediates.noindex/...`:
```
{
  ...
  "dependencies": [
    ... 
    "/Users/mseremet/Library/Developer/Xcode/DerivedData/Crunchyroll-exbviznuyvwvwqbtlgbiywhttpfr/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/SDWebImage.build/module.modulemap"
  ]
}
```

Any idea what causes this?

**Comment by polac24** [comment]
Hmm, that should be replaced here:
https://github.com/spotify/XCRemoteCache/blob/b6318e97855f8f23b2dc670a8765baad80340d25/Sources/XCRemoteCache/Dependencies/DependencyProcessor.swift#L78-L79

I would check your producer and see if build settings `TARGET_TEMP_DIR` is a prefix of that path. Build settings can be inspected in Xcode's Report navigator (CMD+9) -> find build script with postbuild -> extend view with a "hamburger" button on the right.

**Comment by polac24** [comment]
Can you provide more details about your `Podfile`? Is `Vilos` an objc, swift or a mixed target?

I tried to reproduce it (checked `use_modular_headers` and `use_frameworks`) but the modulemap path was always correctly remapped.

**Comment by polac24** [comment]
That should be fixed in #71, available since [0.3.4](https://github.com/spotify/XCRemoteCache/releases/tag/v0.3.4)
